### PR TITLE
Exclude tests from distribution

### PR DESCRIPTION
**Pull request by browniebroke**
## Problem

As explained in #258, this is to exclude the tests from the distributed version of this library.

## Solution

- Move the `tests` folder outside of the main package.
- Exclude them from `find_package()` in setup.py
- Update test command to call `runtests.py` directly (rather than `setup.py test`)
- Updated paths accordingly in test runner and translations generator script

## Commandments

- [x] Write PEP8 compliant code.
- [x] Cover it with tests.
- [x] Update `CHANGES.rst` file to describe the changes, and quote according issue with `GH-<issue_number>`.
- [x] Pay attention to backward compatibility, or if it breaks it, explain why.
- [x] Update documentation (if relevant).

### CODE REVIEW

**Comment by romgar** [comment]
Thanks for the MR @browniebroke ! Did you check that the number of tests running has not changed with your update?

**Comment by browniebroke** [comment]
Good point. Forgot about that.

I compared just now, against the latest build on master (549). I looked at 2 environments: Python 2.7 with Django 1.8 and Python 3.5 with Django 1.10 and they are all running 256 test cases.

File: tox.ini [hunk 1]
```diff
@@ -20,4 +20,4 @@ deps =
     django_trunk: https://github.com/django/django/tarball/master
     freezegun == 0.3.8
 
-commands = coverage run -a setup.py test
```
Comment (romgar) [inline tox.ini]: Any reason to remove the `-a` option?

File: tests/models.py [hunk 2]
```diff
@@ -1,4 +1,4 @@
-from __future__ import unicode_literals
+from __future__ import unicode_literals, absolute_import
```
Comment (romgar) [inline tests/models.py]: Do we really need this `absolute_import`?

Comment (browniebroke) [inline tests/models.py]: Yes, it's failing to import `MutableField` on Python 2.7 without it.

File: tox.ini (hunk 1 above)
Comment (browniebroke) [inline tox.ini]: That's a mistake from myself, will add it back.

**Comment by romgar** [comment]
Thanks @browniebroke !!

**Pull request merged**